app/src/components/features/rules/GettingStarted/WorkspaceOnboarding/OnboardingSteps/TeamWorkspaces/WorkspaceCreation.tsx

```diff
@@ -48,6 +54,7 @@ export const CreateWorkspace: React.FC<Props> = ({ defaultTeamData }) => {
   ]);
 
   const handleCreateNewTeam = () => {
+    trackCreateNewWorkspace("onboarding");
```

Comment:
we already had an event `trackCreateNewTeamClicked`. Please confirm which event we would like to keep.
Moreover, the same event should be called from all the create workspace with different sources. right?